go.mod

```diff
@@ -184,7 +183,10 @@ require (
 	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
 )
 
-require cloud.google.com/go/compute/metadata v0.2.3
+require (
+	cloud.google.com/go/compute/metadata v0.2.3
```

Comment:
Could you add those imports to the big "require" block above, the same that had kingpin before, and `go mod tidy` it again? It's easier if we keep all directs in a single block.

Reply:
Done